introduction: >
  We are going to do a series of exercises to work on your balance,
  cardiovascular fitness, flexibility and strength. 

series_1:
  introduction: >
    Our first series of exercises will work on cardio. You may do them slower
    or faster than the demonstration as you like.
  type: Cardio_Final
  ex1:
    text: "We’ll start with some low punches. These can be done sitting or standing."
    rep: 2
    time_between_rep_in_sec: 3
    video_name: "cardio_punches_low.mp4"
  ex2:
    text: "Let’s do some high punches, you can do these either sitting or standing."
    rep: 1
    time_between_rep_in_sec: 3
    video_name: "cardio_punches_high.mp4"

series_2:
  introduction: >
    Alright User, now we are going to do some balance exercises.
  type: Balance_Final
  ex1:
    text: >
      We’ll start with an easy exercise that just requires you to stand quietly
      with your feet close together.
    rep: 2
    time_between_rep_in_sec: 3
    video_name: "FINAL_feet_together_stance_medium.mp4"

ending: >
  Great job User you’ve completed all the exercises! We’ll do them again in a
  few days.
